modules/oceanbase-ce/src/main/java/org/testcontainers/containers/OceanBaseContainer.java

```diff
@@ -0,0 +1,163 @@
+package org.testcontainers.containers;
+
+import org.apache.commons.lang3.StringUtils;
+import org.testcontainers.utility.DockerImageName;
+
+/**
+ * Testcontainers implementation for OceanBase.
+ * <p>
+ * Supported image: {@code oceanbase/oceanbase-ce}
+ * <p>
+ * Exposed ports:
+ * <ul>
+ *     <li>SQL: 2881</li>
+ *     <li>RPC: 2882</li>
+ * </ul>
+ */
+public class OceanBaseContainer extends JdbcDatabaseContainer<OceanBaseContainer> {
+
+    static final String NAME = "oceanbase";
+
+    static final String DOCKER_IMAGE_NAME = "oceanbase/oceanbase-ce";
+
+    private static final DockerImageName DEFAULT_IMAGE_NAME = DockerImageName.parse(DOCKER_IMAGE_NAME);
+
+    private static final Integer SQL_PORT = 2881;
+    private static final Integer RPC_PORT = 2882;
+
+    private static final String SYSTEM_TENANT_NAME = "sys";
+    private static final String DEFAULT_TEST_TENANT_NAME = "test";
+    private static final String DEFAULT_USERNAME = "root";
+    private static final String DEFAULT_PASSWORD = "";
+    private static final String DEFAULT_DATABASE_NAME = "test";
+
+    private boolean enableFastboot;
+    private String mode;
+    private String tenantName = DEFAULT_TEST_TENANT_NAME;
+    private String driverClassName = "com.mysql.cj.jdbc.Driver";
+
+    public OceanBaseContainer(String dockerImageName) {
+        this(DockerImageName.parse(dockerImageName));
+    }
+
+    public OceanBaseContainer(DockerImageName dockerImageName) {
+        super(dockerImageName);
+        dockerImageName.assertCompatibleWith(DEFAULT_IMAGE_NAME);
+
+        addExposedPorts(SQL_PORT, RPC_PORT);
+    }
+
+    @Override
+    public Integer getMappedPort(int originalPort) {
+        return "host".equals(getNetworkMode()) ? originalPort : super.getMappedPort(originalPort);
+    }
+
+    @Override
+    public String getDriverClassName() {
+        return driverClassName;
+    }
+
+    @Override
+    public String getJdbcUrl() {
+        return getJdbcUrl(DEFAULT_DATABASE_NAME);
+    }
+
+    public String getJdbcUrl(String databaseName) {
+        String additionalUrlParams = constructUrlParameters("?", "&");
+        String prefix = driverClassName.contains("mysql") ? "jdbc:mysql://" : "jdbc:oceanbase://";
```

Comment:
I think we should restrict more here because at first glance if any driverClassName different than mysql will choose oceanbase prefix, and not sure but that would be unexpected.

Reply:
The 'driverClassName' is verified in `withDriverClassName` method, so its value is limited to these two options.